lib/win32/certstore/store_base.rb

```diff
@@ -102,6 +102,29 @@ def cert_delete(store_handler, certificate_thumbprint)
         cert_delete_flag
       end
 
+      # Search certificate from open certificate store and return boolean
+      # store_handler => Open certificate store handler
+      # search_token => CN, RDN or any certificate attribute
+      def cert_search(store_handler, search_token)
+        validate!(search_token)
```

Comment:
The name of this method doesn't convey much about what to expect. I think you're over-optimizing here as this is the only place I see calling this method, and the method itself is a single line of code. I'd bring that line back here.